pkg/flare/archive_dca.go

```diff
@@ -23,6 +24,13 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
+const (
+	HELM_CHART_RELEASE_NAME       = "DD_CHART_RELEASE_NAME"
+	HELM_CHART_RELEASE_NAMESPACE  = "DD_KUBE_RESOURCES_NAMESPACE"
```

Comment:
maybe the name of the envvar should also start by `DD_CHART`

```suggestion
	HELM_CHART_RELEASE_NAMESPACE  = "DD_CHART_RESOURCES_NAMESPACE"
```

Reply:
`DD_KUBE_RESOURCES_NAMESPACE` is already present in the Cluster Agent deployment : https://github.com/DataDog/helm-charts/blob/main/charts/datadog/templates/cluster-agent-deployment.yaml#L240-L241 so used it directly as it does not require any changes and did not want to duplicate an information already available.